packages/office-ui-fabric-react/src/components/Link/Link.types.ts

```diff
@@ -13,7 +13,7 @@ export interface ILink {
   focus(): void;
 }
 
-export interface ILinkProps extends React.AllHTMLAttributes<HTMLAnchorElement | HTMLButtonElement | HTMLElement | LinkBase> {
+export interface ILinkProps extends React.AnchorHTMLAttributes<HTMLAnchorElement | HTMLButtonElement | HTMLElement | LinkBase> {
```

Comment:
Should this be `AnchorHTMLAttributes<...> | ButtonHTMLAttributes<...>`

Reply:
It will be ;)

Reply:
Not sure what happened to your breaking change comment, but technical speaking it would be. If someone added non anchor or button props to their Link, upgrading to this version would break that. I'm guessing this scenario isn't very likely.
That said this isn't super important to me either and it can wait till 7 if you feel that is better.

Reply:
Unfortunately TS doesn't allow me to do `AnchorHTMLAttributes<...> | ButtonHTMLAttributes<...>`

![image](https://user-images.githubusercontent.com/2134901/40908324-aedd978a-67e6-11e8-806e-a0b6056e4160.png)

Reply:
Can you create a `type Foo = React.AnchorHTMLAttributes<HTMLAnchorElement> | React.ButtonHTMLAttributes<HTMLButtonElement>` and then extend off of that?

Reply:
I don't think it's possible to extend of a `type`. But will give it a go.